src/helpers/data/locales.js

```diff
@@ -6,5 +6,9 @@ export default [
   {
     code: 'ru',
     text: 'Русский'
+  },
+  {
+    code: 'it',
+	text: 'Italiano'
```

Comment:
@ZinRicky Please fix indentation here.

Reply:
@ZinRicky Also please place Italian after English, they should come alphabetically.